assimilation_code/modules/assimilation/assim_tools_mod.rst

```diff
@@ -10,25 +10,6 @@ for both mean and spread. In addition, algorithms to do a variety of flavors of
 particle filter, and kernel filters are included. The parallel implementation that allows each observation to update all
 state variables that are close to it at the same time is described in Anderson and Collins, 2007.
 
-Filter types
-------------
-
-Available observation space filter types include:
-
--  1 = EAKF (Ensemble Adjustment Kalman Filter, see Anderson 2001)
--  2 = ENKF (Ensemble Kalman Filter)
--  3 = Kernel filter
--  4 = Observation Space Particle filter
--  5 = Random draw from posterior (contact dart@ucar.edu before using)
--  6 = Deterministic draw from posterior with fixed kurtosis (ditto)
--  7 = Boxcar kernel filter
--  8 = Rank Histogram filter (see Anderson 2010)
--  9 = Particle filter (see Poterjoy 2016)
-
-We recommend using type=1, the EAKF. Note that although the algorithm is expressed in a slightly different form, the
-EAKF is identical to the EnSRF (Ensemble Square Root Filter) described by Whitaker and Hamill in 2002. Highly
-non-gaussian distributions may get better results from type=8, Rank Histogram filter.
```

Comment:
Does this note on EAKF vs EnSRF need adding to the QCEFF filter option docs:
https://docs.dart.ucar.edu/en/quantile_methods/guide/qceff_probit.html#available-filter-kinds ?

Reply:
discussed with jla
can pul this EnSRF comment out.